src/yetibot/core/adapters/slack.clj

```diff
@@ -234,6 +234,9 @@
           yetibot? (= yetibot-uid (:user event))
           user-model (assoc (users/get-user cs (:user event))
                             :yetibot? yetibot?)
+          ;; TODO - call (a/resolve-users event) to ensure users are resolved,
+          ;; then pass the resulting map along to `handle-raw` as
+          ;; `:resolved-users` below in addition to `:body`.
```

Comment:
This is a potential starting place. 

1. Add `a/resolve-users` function
2. call it via `(a/resolve-users adapter event)`
3. in `a/resolve-users` log the `event` and start looking at how we can pull users out of it and turn them into canonical persistent records